### PR TITLE
MRG: Add distances plot to report 

### PR DESCRIPTION
**Pull request by doutib**
Addresses issue #164

### CODE REVIEW

**Comment by doutib** [comment]
@jarrodmillman, the plot does not seem to be displayed in the report. After checking the code several times, I cannot see whether this is an error from me or not.
Can you check very quickly and tell me if there is anything is wrong please?

**Comment by doutib** [comment]
@shamindras do we need to compile the .rst files, or github does it for us?
This one does not display my plot.

File: doc/source/report/plots/plot_distances.py [hunk 1]
```diff
@@ -7,27 +8,17 @@
...
-dist_strain = []
-strain = 0
-dist = distances_bystrain(strain, step=step)
-while dist.size > 0:
-    dist_strain.append(dist[dist >= threshold])
-    if verbose:
-        print('strain %s loaded.' % strain)
-    strain += 1
-    dist = distances_bystrain(strain, step=step)
+dist_strain = list(np.load('report/plots/distances.npy'))
```
Comment (doutib) [inline doc/source/report/plots/plot_distances.py]: @jarrodmillman, one question: 
- Is it pointing to the right file or should I change the path? 
The plots are still not displayed in the rst file.

**Comment by doutib** [comment]
@jarrodmillman 
Will you be able to help dealing with this issue?

**Comment by doutib** [comment]
I have finally added figures to the figure directory and included them in the rst document. For some reasons the plots are not compiling.
I kept the script and the data in the plot dir, in case we want to see how it is generated.

I believe we are able to merge now.

**Comment by jarrodmillman** [comment]
@doutib Sorry I missed your earlier question.  I am just getting caught up on things now.

I merged another pull request since you last rebased.  Please rebase and I will merge this.

By the way, how did you try to compile the plots?  You should just be able to type
```
$ cd doc/source/
$ make html
```
to generate the HTML page and
```
$ cd doc/source/
$ make latexpdf
```
to generate the PDF.

Both the commands will generate the plots using the plot directive.  You can see the code I use to do that here:
```
$ cat _sphinxext/plot_directive.py
```

Is this what you were doing?  If so, what error message did you get?

You don't need to change anything (i.e., other than rebasing this), I am just curious.

**Comment by doutib** [comment]
@jarrodmillman No worries, I tried with the make html command and nothing appears on the document excepted for a "Source code" link which provides me the python code I use to generate the plot...

I ll rebase asap.

**Comment by doutib** [comment]
@jarrodmillman rebasing done.

**Comment by jarrodmillman** [comment]
@doutib I am not sure what happened, but something is weird about your "rebase".  It added 2 commits from HongfeiZhao.  It also didn't update you with master.  " This branch is 9 commits ahead, 10 commits behind berkeley-stat222:master" according to:
 https://github.com/doutib/mousestyles/tree/contributing-issue-164

Anyway, I cherry picked your commits and created a new pull request #187.  Once the tests pass I will merge that one instead of this one to keep the history nice.

**Pull request closed without merging**